code/modules/mob/mob.dm

```diff
@@ -1506,3 +1506,6 @@ mob/proc/assess_threat()
 
 mob/proc/walking()
 	return !(lying || flying)
+
+/mob/proc/dexterity_check()
```

Comment:
Just as something I noticed this would cause : ghosts wouldn't be able to work machines.
Did you test this as an admin ghost?